### PR TITLE
Ports all protocol pages of sirius-biz to Tycho.

### PR DESCRIPTION
**Pull request by andyHa**
grafik
grafik
grafik
grafik
grafik
grafik

### CODE REVIEW

File: src/main/resources/default/templates/biz/protocol/mail.html.pasta [hunk 1]
```diff
@@ -10,42 +10,79 @@
...
-        <a href="/system/mails" class="btn"><i class="icon-chevron-left"></i> @i18n("NLS.back")</a>
+    <div class="card mb-4">
+        <div class="card-body">
+            <h5 class="card-title">@i18n("MailProtocol.html")</h5>
+            <pre class="mt-4 mb-0" style="max-height: 400px; overflow-y: auto">@mail.getHtmlContent()</pre>
+        </div>
     </div>
-</w:page>
+
+    <t:formBar btnLabel="" />
```
Comment (cschierle) [inline src/main/resources/default/templates/biz/protocol/mail.html.pasta:87]:
```suggestion
    <t:formBar btnLabel=""/>

File: src/main/resources/biz_de.properties [hunk 2]
```diff
@@ -484,10 +484,13 @@ MailProtocol.receiver = Empfänger
 MailProtocol.receiverName = Empfängername
 MailProtocol.sender = Absender
 MailProtocol.senderName = Absendername
-MailProtocol.success = Erfolgreich
+MailProtocol.failed = fehlgeschlagen
+MailProtocol.successful = erfolgreich
```
Comment (cschierle) [inline src/main/resources/biz_de.properties:488]:
```suggestion
MailProtocol.failed = Fehlgeschlagen
MailProtocol.successful = Erfolgreich

File: src/main/resources/default/templates/biz/protocol/error.html.pasta [hunk 3]
```diff
@@ -62,20 +77,16 @@
...
-        <a href="/system/errors" class="btn"><i class="icon-chevron-left"></i> @i18n("NLS.back")</a>
+    <div class="card mb-4">
+        <div class="card-body">
+            <h5 class="card-title">@i18n("StoredIncident.stack")</h5>
+            <pre class="mt-4 mb-4">@incident.getStack()</pre>
+        </div>
     </div>
 
-</w:page>
+    <t:formBar btnLabel="" />
```
Comment (cschierle) [inline src/main/resources/default/templates/biz/protocol/error.html.pasta:91]:
```suggestion
    <t:formBar btnLabel=""/>

File: src/main/resources/default/templates/biz/protocol/logs.html.pasta [hunk 4]
```diff
@@ -1,64 +1,64 @@
...
-                        </td>
-                    </tr>
-                </i:for>
-                </tbody>
-            </w:table>
+                            </th>
+                        </tr>
+                        </thead>
+                        <tbody>
+                        <i:for type="sirius.biz.protocol.LoggedMessage" var="msg" items="logs.getItems()">
```
Comment (sabieber) [inline src/main/resources/default/templates/biz/protocol/logs.html.pasta:37]: Optional: Rename msg to something more easily readable

File: src/main/resources/biz_de.properties (hunk 2 above)
Comment (sabieber) [inline src/main/resources/biz_de.properties:488]: Would start these uppercase, like all other Translation Texts

**Review by cschierle — APPROVED** [review]

File: src/main/resources/default/templates/biz/protocol/protocol.html.pasta [hunk 5]
```diff
@@ -1,80 +1,85 @@
...
-                        </td>
-                    </tr>
-                </i:for>
-                </tbody>
-            </w:table>
+                            </th>
+                        </tr>
+                        </thead>
+                        <tbody>
+                        <i:for type="sirius.biz.protocol.JournalEntry" var="msg" items="protocol.getItems()">
```
Comment (sabieber) [inline src/main/resources/default/templates/biz/protocol/protocol.html.pasta:42]: Optional: Rename msg to something more fitting

File: src/main/resources/default/templates/biz/protocol/audit_logs.html.pasta [hunk 6]
```diff
@@ -1,64 +1,66 @@
...
-                            </div>
-                            <pre class="plain" style="white-space: break-spaces">@i18n(msg.getMessage())</pre>
-                        </td>
-                    </tr>
-                </i:for>
+                            </th>
+                        </tr>
+                        </thead>
+                        <tbody>
+                        <i:for type="sirius.biz.protocol.AuditLogEntry" var="msg" items="messages.getItems()">
```
Comment (sabieber) [inline src/main/resources/default/templates/biz/protocol/audit_logs.html.pasta:40]: Optional: Rename msg to something more fitting

**Review by sabieber — APPROVED** [review]

**Review by sabieber — APPROVED** [review]

**Review by qw3ry — APPROVED** [review]

**Review by mkeckmkeck — APPROVED** [review]

**Pull request merged**